tools/SetupFlow/DevHome.SetupFlow/ViewModels/ReviewViewModel.cs

```diff
@@ -138,5 +143,44 @@ private async Task OnSetUpAsync()
         {
             _log.Error($"Failed to initialize elevated process.", e);
         }
-   }
+    }
+
+    [RelayCommand(CanExecute = nameof(HasDSCTasksToDownload))]
+    private async Task DownloadConfigurationAsync()
+    {
+        try
+        {
+            // Supported save file types
+            var wingetFileType = ".winget";

```

Comment:
PR #2459 also declares these two strings as valid file types.  Is it possible to move `wingetFileType` and `dscFileType` to a common location?

Reply:
We can consider this enhancement after https://github.com/microsoft/devhome/pull/2492 PR is merged as currently the file dialog takes the file types as a combined pattern string. `DesiredStateConfiguration` service may be a good canddidate for storing this information (likely in a collection of strings).